xyz

```diff
@@ -66,7 +66,7 @@ Options:
 "
 
 abort() {
-  printf "%s\n" "$1" >&2
+  printf '%s\n' "$1" >&2
```

Comment:
```bash
  printf "%s\n" "$1" >&2
            ^-- SC1117
```

[SC1117][1]: Backslash is literal in `"\n"`. Prefer explicit escaping: `"\\n"`.


[1]: https://github.com/koalaman/shellcheck/wiki/SC1117